docs/deployment-guide/goals/production/README.md

```diff
@@ -19,7 +19,7 @@ operator company.
 
 - Each configured Dataspace Deployment Environment must have a running sovity Keycloak DAPS.
 - Each configured Dataspace Deployment Environment must have a running Logging House.
-- To make use of the Data Catalog, each configured Dataspace Deployment Environment must have a running EDC connector with the Broker Crawler Extension.
+- To make use of the Data Catalog, each configured Dataspace Deployment Environment must have configured a running Catalog Crawler (based on the sovity EDC Connector).
```

Comment:
the "based on the sovity EDC Connector" part makes no sense from the reader's perspective